tests/host_tools/drive.py

```diff
@@ -19,6 +20,14 @@ def __init__(self, path: str, size: int = 256, fs_format: str = "ext4"):
         Raises if the file system format is not supported, if the file already
         exists, or if it ends in '/'.
         """
+
+        # If no path is supplied, use a temporary file.
+        # This is useful to force placing the file on disk, not in memory,
+        # because qemu vhost-user-blk backend always uses O_DIRECT,
+        # but O_DIRECT is not supported by tmpfs.
+        if path is None:
```

Comment:
It is worth making the `path` parameter in `__init__` optional, like: `path: str = None`

Reply:
true, done, thanks